src/main/resources/spam_words.txt

```diff
@@ -0,0 +1 @@
+Canonical Truelancer
```

Comment:
Let each word be on a separate line, makes it easier to quickly parse and understand what we are looking for